docs/includes/file-get-started-excel-vue.md

```diff
@@ -16,131 +18,64 @@ In this article, you'll walk through the process of building an Excel add-in usi
     npm install -g yo generator-office
     ```
 
-## Generate a new Vue app
-
-Use the Vue CLI to generate a new Vue app. From the terminal, run the following command and then answer the prompts as described below.
-
-```bash
-vue init webpack my-add-in
-```
-
-When responding to the prompts that are generated by the previous command, override the default answers for the following 3 prompts. You can accept the default answers for all other prompts.
-
-- **Install vue-router?** `No`
-- **Set up unit tests:** `No`
-- **Setup e2e tests with Nightwatch?** `No`
+### Create the web app
 
-![Vue CLI prompts](../images/vue-cli-prompts.png)
+1. Create a folder on your local drive and name it **my-addin**. This is where you'll create the files for your app.
 
-## Generate the manifest file
-
-Each add-in requires a manifest file to define its settings and capabilities.
-
-1. Navigate to your app folder.
+2. Navigate to your app folder.
 
     ```bash
-    cd my-add-in
+    cd my-addin
     ```
 
-2. Use the Yeoman generator to generate the manifest file for your add-in. Run the following command and then answer the prompts as shown below.
+3. Use the Yeoman generator to generate the manifest file for your add-in. Run the following command and then answer the prompts as shown in the following screenshot.
```

Comment:
Can we change "as shown in the following screenshot" to "with the following choices"? We're describing the prompt answers in both text and the screenshot.